scientific/E3SM_DV3D_Native_Grid_Test/E3SM_DV3D_ne30_Native_Grid_Test.ipynb

```diff
@@ -71,7 +71,7 @@
     "\n",
     "The data file for this tutorial is \"20180129.DECKv1b_piControl.ne30_oEC.edison.cam.h0.0500-01.nc\" from the CMIP6 standard resolution (ne30) piControl run. \n",
     "\n",
-    "The grid file used is \"ne30np4_latlon.091226.nc\". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the \"ne30np4_latlon.091226.nc\" file which should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.\n",
+    "The grid file used is \"ne30np4_latlon.091226.nc\". To download this grid file, go to https://portal.nersc.gov/project/acme/hdavis/ and click on the \"ne30np4_latlon.091226.nc\" file whcih should automatically download via your browser. To facilitate running this notebook, the CMIP6 data file can also be downloaded from this same location.\n",
```

Comment:
Same spelling error 'whcih' should be 'which'

Reply:
Yes, it should be fixed - thanks!  I pushed a commit to fix the typo + some other minor edits.